app/server/lib/ActiveDoc.ts

```diff
@@ -1293,8 +1293,8 @@ export class ActiveDoc extends EventEmitter {
    * @param {String} tableId: The string identifier of the table.
    * @returns {Promise} Promise for a string representing the generated table schema.
    */
-  public async fetchTableSchema(docSession: DocSession): Promise<string> {
-    this._log.info(docSession, "fetchTableSchema(%s)", docSession);
+  public async fetchPythonCode(docSession: DocSession): Promise<string> {
```

Comment:
Could you also update the description in the JSDoc comment above and remove the `tableId` param? Thanks.